includes/admin/meta-boxes/views/html-product-data-general.php

```diff
@@ -31,8 +31,6 @@
 				'description' => __( 'This text will be shown on the button linking to the external product.', 'woocommerce' ),
 			)
 		);
-		
-		do_action( 'woocommerce_product_options_external' );
```

Comment:
We actually cannot remove existing actions for backward compatibility. This would need to be added back.